### PR TITLE
Improve app.js

### PR DESCRIPTION
**Pull request by ztplz**

### CODE REVIEW

File: website/app.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ function generate(
     size: {
       height: 300,
       // @ts-ignore
```
Comment (kitsonk) [inline website/app.js]: This can be removed now with the below change.

Comment (ztplz) [reply website/app.js]: I removed it just now, but it resulted in a red CI.

File: website/app.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ function generate(
...
-      width: window.chartWidth || 375 // TODO: do not use global variable
+      width: document.title === "deno" ? 375 : 800
```
Comment (ry) [inline website/app.js]: I don't think it's a good idea to change the layout based on title. We need a more principled approach.

**Comment by ry** [comment]
Thank you - but closing this one. Let's discuss on gitter if there's something else that can be done.

**Pull request closed without merging**